diagnostics.yml

```diff
@@ -142,7 +142,7 @@ stages:
       parameters:
         name: Alpine3_13
         osGroup: Linux
-        dockerImage: mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.13-WithNode-20210719120822-9b5bbc2
+        dockerImage: mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.13-20210719120822-69a32fe
```

Comment:
This is causing CI to fail to initialize the container I think.  Locally this image doesn't build sos where the WithNode did:
```
Invoking "/home/mikem/diagnostics/eng/gen-buildsys-clang.sh" "/home/mikem/diagnostics" 3 "9" x64 /home/mikem/diagnostics/eng Debug   -DCLR_MANAGED_BINARY_DIR=/home/mikem/diagnostics/artifacts/bin -DCLR_BUILD_TYPE=Debug
Unable to find Clang Compiler
```


But either image is going to have problems because now on my local machine/build, libsos.so is missing libintl symbols:
 ```
ldd /home/mikem/diagnostics/artifacts/bin/dotnet-dump/Debug/netcoreapp2.1/publish/linux-musl-x64/libsos.so
        /lib/ld-musl-x86_64.so.1 (0x7ff8597ff000)
        libgcc_s.so.1 => /usr/lib/libgcc_s.so.1 (0x7ff8594d8000)
        libstdc++.so.6 => /usr/lib/libstdc++.so.6 (0x7ff859336000)
        libc.musl-x86_64.so.1 => /lib/ld-musl-x86_64.so.1 (0x7ff8597ff000)
Error relocating /home/mikem/diagnostics/artifacts/bin/dotnet-dump/Debug/netcoreapp2.1/publish/linux-musl-x64/libsos.so: libintl_dgettext: symbol not found
Error relocating /home/mikem/diagnostics/artifacts/bin/dotnet-dump/Debug/netcoreapp2.1/publish/linux-musl-x64/libsos.so: libintl_bindtextdomain: symbol not found
```
I'm looking at what diagnostics repo PAL code requires libintl on Alpine (or in general) where the runtime PAL doesn't.  In a separate test branch, I'm clean up the diagnostics PAL code that is no longer in the runtime PAL.

Reply:
We should probably update the detection scripts. I'll try to carve some time today

Reply:
I've figured the agent initialization issue.

Reply:
I have a couple other fixes. But this issue requires us to rebuild the image. I will open a PR in prereqs and then push my other fixes.